data/wanderer/wanderers.txt

```diff
@@ -876,8 +876,11 @@ event "wanderers: more systems lost"
 		spaceport `The raft of algae that supports the spaceport is probably at least a dozen meters thick, but flexible enough that it bends as the ocean swells pass underneath, causing individual buildings to rise up or tilt slightly relative to their neighbors. Now that this world has been captured by the Unfettered, nearly all the buildings are empty; it appears that the Hai are not too fond of the idea of inhabiting a water world, so they are using it mostly as a military base.`
 
 planet "The Eye"
-	description ""
 	spaceport ""
```

Comment:
Maybe add a newline here? (Probably not needed for the parser, but it just looks a little bit nicer.)